src/main/resources/org/sonar/l10n/checkstyle/rules/checkstyle/com.puppycrawl.tools.checkstyle.checks.coding.UnnecessarySemicolonAfterTypeMemberDeclarationCheck.html

```diff
@@ -1,3 +1,3 @@
 <p>
-    Checks if unnecessary semicolon is used after type member declaration.
+    Checks if unnecessary semicolon is used after type declaration.
```

Comment:
Based on check names, this looks mixed up.

Reply:
yes, this is totally right. it was the wrong file... thanks and fixed. :)